### PR TITLE
Add ja_jp.json for Japanese

### PR DESCRIPTION
**Pull request by Toshimichi0915**
Don't call me PR farmer please

### CODE REVIEW

**Review by modmuss50 — APPROVED** [review]

File: enigma/src/main/resources/lang/ja_jp.json [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+	"progress.mappings.enigma_directory.loading": "マッピングファイルの読込中",
+	"progress.mappings.enigma_directory.writing": "クラスの書き出し中",
+	"progress.mappings.tiny_file.loading": "マッピングファイルの読込中",
+	"progress.mappings.tiny_v2.loading": "マッピングファイルの読込中",
+	"progress.mappings.srg_file.generating": "マッピングの作成中",
+	"progress.mappings.srg_file.writing": "マッピングの書き出し中",
+	"progress.stats": "統計情報を作成中",
+	"progress.stats.data": "データの作成中",
+
+	"javadocs.edit": "Javadocsを編集する",
```
Comment (reo-ar) [inline enigma/src/main/resources/lang/ja_jp.json:135]: Javadocsを編集する -> Javadocを編集する

File: enigma/src/main/resources/lang/ja_jp.json [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+	"javadocs.instruction": "このJavadocsを編集する",
```
Comment (reo-ar) [inline enigma/src/main/resources/lang/ja_jp.json:136]: このJavadocsを編集する -> このJavadocを編集する

File: enigma/src/main/resources/lang/ja_jp.json [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+	"disconnect.disconnected": "切断されました",
+	"disconnect.server_closed": "サーバーが終了しました",
+	"disconnect.wrong_jar": "Jarのチェックサムが一致しません(間違ったJarです)!",
+	"disconnect.wrong_password": "パスワードが違います",
+	"disconnect.username_taken": "ユーザー名はすでに使われています",
+
+	"message.chat.text": "%s: %s",
+	"message.connect.text": "[+] %s",
+	"message.disconnect.text": "[-] %s",
+	"message.edit_docs.text": "%s が %s のJavadocsを編集しました",
```
Comment (reo-ar) [inline enigma/src/main/resources/lang/ja_jp.json:172]: のJavadocsを編集しました -> のJavadocを編集しました

File: enigma/src/main/resources/lang/ja_jp.json [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+	"menu.file.reload_mappings": "マッピングを再読み込みする",
+	"menu.file.reload_all": "Jarもしくはマッピングを再読み込みする",
+	"menu.file.export.source": "ソースコードをエクスポートする...",
+	"menu.file.export.jar": "Jarをエクスポートする...",
+	"menu.file.stats": "マッピングの統計情報...",
+	"menu.file.stats.title": "マッピングの統計情報",
+	"menu.file.stats.top_level_package": "トップレベルパッケージ:",
+	"menu.file.stats.synthetic_parameters": "合成パラメータを含める",
+	"menu.file.stats.generate": "図を生成する",
+	"menu.file.exit": "終了",
```
Comment (reo-ar) [inline enigma/src/main/resources/lang/ja_jp.json:33]: File Exit -> ファイルを閉じる

It's more of a close than an exit program it's an exit file more commonly found as "ファイルを閉じる" in programs

**Review by reo-ar — CHANGES_REQUESTED** [review]

File: enigma/src/main/resources/lang/ja_jp.json (hunk 4 above)
Comment (reo-ar) [reply enigma/src/main/resources/lang/ja_jp.json:33]: Edit: I'm not sure about the context of this, if this refers to exit program then "終了" is correct.

Comment (Toshimichi0915) [reply enigma/src/main/resources/lang/ja_jp.json:33]: that message is used for the exit button of the program.
This doesn't need changing.

Comment (2xsaiko) [reply enigma/src/main/resources/lang/ja_jp.json:33]: This is actually 'Exit' in the File menu which exits the program

**Review by 2xsaiko — APPROVED** [review]

**Comment by 2xsaiko** [comment]
~~PR farmer~~

Thanks!

**Pull request merged**